sources/ca/ab/city_of_lethbridge.json

```
@@ -12,6 +12,8 @@
         }
     },
     "data": "http://gis.lethbridge.ca/lethwebgisarcgis/rest/services/OpenData/odl_parcels/MapServer/0",
+    "license": {"url": "https://www.lethbridge.ca/Pages/OpenDataLicense.aspx",
+               "attribution":false},
```

Please follow the existing JSON formatting on these.

```suggestion
    "license": {
        "url": "https://www.lethbridge.ca/Pages/OpenDataLicense.aspx",
        "attribution": false
    },
```

---

Sure - I'll make an effort to make sure I'm following the right format. For my own clarification, when do you use 
license : {url: ..., attribution: ...}, 
and when do you use 
license: licenseurl
attribution: ...

That is, when is license its own dictionary, and when is it not?

---

You can read more about it here: https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md#optional-tags

The URL-only `license` attribute is supported but deprecated. Ideally it would always be a dictionary.

---

Okay, sounds good. Do you use the attribution name field to include text like "Contains information licensed under..." or do you simply flag it as attribution required?